dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/servlet/DhisWebApiWebAppInitializer.java

```diff
@@ -127,8 +127,10 @@ public static void setupServlets(
         context.addFilter("openSessionInViewFilter", OpenEntityManagerInViewFilter.class);
     openSessionInViewFilter.setInitParameter(
         "entityManagerFactoryBeanName", "entityManagerFactory");
-    openSessionInViewFilter.addMappingForUrlPatterns(null, false, "/*");
-    openSessionInViewFilter.addMappingForServletNames(null, false, "dispatcher");
+    openSessionInViewFilter.addMappingForUrlPatterns(
+        EnumSet.of(DispatcherType.REQUEST, DispatcherType.ASYNC), false, "/*");
```

Comment:
Does this mean now any URL will be handled as if it is async? Might that have unintended consequences?